### PR TITLE
 I JUST CONFLICTED EVERYTHING

### PR DESCRIPTION
**Pull request by Aurorablade**
Okay so many may recall i did PR to do some case consistency a while back. Didn't fix the underlying issue.

I JUST WENT AHEAD AND NUKED EVERYTHING FROM ORBIT.

All Sounds and when i can get to them DMI files are now lower case.
You are going to note one web edit delete cuase me and github and some names has an issue and i had to go around MY ASS  to fix it while screaming.

So why do this?
We need all file paths (sounds/dmi) refs to be consistent for some server based compileing stuff. Alf may be able to explain this a bit better as i simply will start throwing the f-bomb every other word right now.
HOPEFULLY, I can get away without just recasing the DMI files on a base level and not have to like..dig into them.

This is alot of changes...Sari is now having a anurysm...I think the github just exploded.

One more note going FORWARD we will most likely being needing to enforce the caseing via travis.

### CODE REVIEW

File: code/datums/spells/horsemask.dm [hunk 1]
```diff
@@ -45,4 +45,4 @@
 		qdel(target.wear_mask)
 	target.equip_to_slot_if_possible(magichead, slot_wear_mask, 1, 1)
 
-	target.flash_eyes()
+	target.flash_eyes()
```
Comment (Aurorablade) [inline code/datums/spells/horsemask.dm]: WHAT EVEN CAUSED THIS ONE

File: code/game/gamemodes/gameticker.dm [hunk 2]
```diff
@@ -43,9 +43,9 @@ var/round_start_time = 0
 	login_music = pick(\
 	'sound/music/THUNDERDOME.ogg',\
```
Comment (Aurorablade) [inline code/game/gamemodes/gameticker.dm]: HOLY FUCK HOW DID I MISS THAT ONE

**Comment by Aurorablade** [comment]
So i am gonna need to back check as i am sure i missed something and i need to double check file pathings i note that i did change a path /ai/ to /AI/

**Comment by Aurorablade** [comment]
I don't even know what this counts as...

**Comment by KasparoVy** [comment]
https://www.youtube.com/watch?v=ydxIeO-2C2M

**Comment by Allfd** [comment]
This is hopefully a fix to the cache issues we have been having...

PSA:  FooBar.dmi != Foobar.dmi

**Comment by KasparoVy** [comment]
Wow, at long last! Gj @aurorablade

**Pull request merged**